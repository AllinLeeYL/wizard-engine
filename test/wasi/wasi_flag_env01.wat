;; A test case to confirm the behavior of environment related flags.
;; Therefore, wasm file itself is almost empty.

;; env01: Error Case - wizeng should not accept a environment variable using --env flag which doesn't contain an equal symbol (e.g., FOO)
(module
  (func $main (export "main")
  )
)
