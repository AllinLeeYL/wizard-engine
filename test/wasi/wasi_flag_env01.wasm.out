wizeng: unrecognized option "--env=FOO"
